service/src/main/resources/application.yml

```diff
@@ -13,6 +13,14 @@ spring:
   # https://github.com/DataBiosphere/terra-common-lib/blob/480ab3daae282ddff0fef8dc329494a4422e32f1/src/main/java/bio/terra/common/logging/GoogleJsonLayout.java#L118
   application.name: cbas
   application.version: ${cbas.version.gitHash:unknown}
+  datasource:
+    jdbc-url: jdbc:postgresql://${DATABASE_HOSTNAME:localhost}:5432/${DATABASE_NAME:cbas_db}
+    username: ${DATABASE_USER:cbas_user}
+    password: ${DATABASE_USER_PASSWORD:cbas_password}
+  sql:
+    init:
+      mode: always
```

Comment:
Is this now making the other `cbas-database` (and maybe even `liquibase` too?) section redundant? If so, it might be nice to tidy them up. Right now we have 3 copy/pastes of the same set of database connectivity information within 30 lines of each other 😬 !

Reply:
I decided to keep the `cbas-database` config. There is a `CbasDatabaseConfguration.java` file that exists that uses this set up for its `@ConfigurationProperties`. Trying to point to the existing prefix `spring.datasource` caused a duplication error, so for now I opted to keep 2 copies of the database code. There is an option to add an `@Profile` annotation, but it seems that I'd still have to duplicate code in going that route.

Reply:
Ok. If we can get some time with a spring expert, it might be nice to try to resolve some of these, but for now, duplicating isn't the worst thing in the world I guess

Reply:
I assume removing the liquibase specific one didn't cause any problems? That's a good sign at least!